Doc/library/decimal.rst

```diff
@@ -790,7 +790,7 @@ Decimal objects
       the current thread's context is used.
 
       An error is returned whenever the resulting exponent is greater than
-      :attr:`Emax` or less than :attr:`Etiny`.
+      :attr:`Emax` or less than :meth:`Etiny`.
```

Comment:
```suggestion
      :attr:`Context.Emax` or less than :meth:`Context.Etiny`.
```

Oops, I missed a spot here—this fixes one warning and makes another correct once documented.